kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -215,34 +216,48 @@ impl CodegenBackend for GotocCodegenBackend {
     /// Types such as `bin`, `cdylib`, `dylib` will trigger the native linker.
     ///
     /// Thus, we manually build the rlib file including only the `rmeta` file.
+    ///
+    /// For cases where no metadata file was requested, we stub the file requested by writing the
+    /// path of the `metadata.json` file so `kani-driver` can safely find the latest metadata.
```

Comment:
Too many metadatas :) So just clarify this a bit:
```suggestion
    /// path of the `kani-metadata.json` file so `kani-driver` can safely find the latest metadata.
```